src/Microsoft.DotNet.Tools.Compiler.Native/ILCompilerInvoker.cs

```diff
@@ -54,7 +46,8 @@ private void InitializeArgs(NativeCompileSettings config)
             argsList.Add($"\"{inputFilePath}\"");
             
             // System.Private.CoreLib Reference
-            var coreLibPath = Path.Combine(config.IlcPath, OSCoreLibNameMap[config.OS]);
+            String[] coreLibs = new String[] { "System.Private.CoreLib.dll", "System.Private.Corelib.dll" };
+            var coreLibPath = Path.Combine(config.IlcPath, Array.Find(coreLibs, lib => File.Exists(Path.Combine(config.IlcPath, lib))));
```

Comment:
Why not?

Checkpoint CLI as is. Call it Stable.
Update CoreRT CI to use Stable CLI instead of Latest CLI.
Upload new package format into the ILC feed.
Check in a breaking change to CLI Latest that picks up from ILC feed. Wait for drop.
Make changes in CoreRT and update CoreRT CI to use Latest.